tools/server/webui/src/lib/components/app/chat/ChatMessages/ChatMessageAssistant.svelte

```diff
@@ -76,9 +79,17 @@
 		shouldBranchAfterEdit = false,
 		siblingInfo = null,
 		textareaElement = $bindable(),
-		thinkingContent
+		thinkingContent,
+		toolCallContent = null
 	}: Props = $props();
 
+	const parsedToolCalls = $derived(() =>
+		Array.isArray(toolCallContent) ? (toolCallContent as ApiChatCompletionToolCall[]) : null
+	);
+	const fallbackToolCallContent = $derived(() =>
+		typeof toolCallContent === 'string' ? toolCallContent : null
+	);
```

Comment:
Here we can use simpler expression without calling a function (which then should be called by `$derived.by` btw).

```suggestion
	const toolCalls = $derived(Array.isArray(toolCallContent) ? (toolCallContent as ApiChatCompletionToolCall[]) : null);
	const fallbackToolCalls = $derived(typeof toolCallContent === 'string' ? toolCallContent : null);
```